panel/tests/test_util.py

```diff
@@ -87,16 +84,6 @@ def test_abbreviated_repr_list():
     assert abbreviated_repr(['some really, really long string']) == "['some really, ...]"
 
 
-def test_abbreviated_repr_ordereddict():
```

Comment:
I think we should still support OrderedDict, even though the code base does not use it.